### PR TITLE
OCPBUGS-26757: Change KAS bootstrap image to cluster-config-api

### PR DESCRIPTION
**Pull request by Patryk-Stefanski**
**What this PR does / why we need it**: 
Change KAS bootstrap image to cluster-config-api.

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes # [OCPBUGS-26757](https://issues.redhat.com/browse/OCPBUGS-26757)

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

**Comment by sjenning** [comment]
There is also this :frowning: 
https://github.com/openshift/hypershift/blob/main/ignition-server/controllers/local_ignitionprovider.go#L241-L323

File: ignition-server/controllers/local_ignitionprovider.go [hunk 1]
```diff
@@ -258,18 +258,18 @@ func (p *LocalIgnitionProvider) GetPayload(ctx context.Context, releaseImage str
 			}
 		}
 
-		component = "cluster-config-operator"
-		clusterConfigOperatorImage, ok := imageProvider.ImageExist(component)
+		component = "cluster-config-api"
```
Comment (sjenning) [inline ignition-server/controllers/local_ignitionprovider.go:261]: This is used as both the image name and binary name, but they are different for the cluster-config-api image.

**Comment by sjenning** [comment]
```
error getting ignition payload: failed to download binaries: failed to extract image file: failed to extract image file: file not found
```
in the ignition-server logs due to ^

File: ignition-server/controllers/local_ignitionprovider.go (hunk 1 above)
Comment (Patryk-Stefanski) [reply ignition-server/controllers/local_ignitionprovider.go:261]: Good spot, thanks. Adressed by ammending the commit.

**Pull request merged**